compiler/ballerina-backend-jvm/src/main/ballerina/src/compiler_backend_jvm/jvm_error_gen.bal

```diff
@@ -59,10 +59,38 @@ type ErrorHandlerGenerator object {
         }
     }
 
-    function generateCatchInsForTrap(bir:ErrorEntry currentEE, jvm:Label endLabel,
-                                    jvm:Label errorValueLabel, jvm:Label otherErrorLabel, jvm:Label jumpLabel) {
+    function generateCatchInsForTrap(bir:Function func, bir:ErrorEntry currentEE, jvm:Label startLabel, jvm:Label endLabel, jvm:Label jumpLabel, 
+    						InstructionGenerator instGen, TerminatorGenerator termGen) {
         self.mv.visitLabel(endLabel);
         self.mv.visitJumpInsn(GOTO, jumpLabel);
+	    if (currentEE is JErrorEntry) {
```

Comment:
formatting issue in this whole section. looks like it does not use 4 columns space?